bootloader/bootloader.go

```diff
@@ -163,6 +166,30 @@ func genericSetBootConfig(systemFile string, content []byte) (bool, error) {
 	return true, osutil.AtomicWriteFile(systemFile, content, 0644, 0)
 }
 
+func genericUpdateBootConfigFromAssets(systemFile string, assetName string) error {
+	currentBootConfigEdition, err := editionFromDiskConfigAsset(systemFile)
+	if err != nil && err != errNoEdition {
+		return err
+	}
+	if err == errNoEdition {
+		return nil
+	}
+	newBootConfig := assets.Internal(assetName)
+	if len(newBootConfig) == 0 {
+		return fmt.Errorf("no boot config asset with name %q", assetName)
+	}
+	gbs, err := configAssetFrom(newBootConfig)
```

Comment:
What does `gbs` stand for?

Reply:
grub boot script, we later agreed on using boot config, I may have missed some spots when renaming